classes/hook_callbacks.php

```diff
@@ -43,6 +44,19 @@ public static function before_standard_top_of_body_html_generation(before_standa
         $hook->add_html(envbarlib::get_inject_code());
     }
 
+    /**
+     * This is the hook enables the plugin to add one or more menu item.
+     *
+     * @param extend_user_menu $hook
+     */
+    public static function extend_user_menu(extend_user_menu $hook): void {
+        // Get items to add.
```

Comment:
need to only show this for site admins on production

we can show this to anyone on non-prod